src/card.css

```diff
@@ -0,0 +1,89 @@
+.store {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-right: 50px;
+  padding: 32px 16px 16px;
+}
+
+.store__desc {
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 40px 0 4px;
+  font-weight: 500;
+  width: 166px;
+}
+
+.store__number {
+  font-size: 10px;
+  color: #616070;
+  margin: 0;
+}
+
+.store__icon {
+  display: flex;
+  justify-content: center;
+}
+
+.store__icon-photo {
+  width: 160px;
+  height: 134px;
+}
+
+.store__rate {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 16px 0 24px;
+  height: 16px;
+}
+
+.store__rate-info {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: #060b35;
+  margin: 0;
```

Comment:
It's better to set only margin-bottom or margin-top for all elements of the card